src/garage/torch/policies/gaussian_mlp_policy.py

```diff
@@ -115,6 +115,22 @@ def get_action(self, observation):
                     dict(mean=dist.mean.squeeze(0).numpy(),
                          log_std=(dist.variance**.5).log().squeeze(0).numpy()))
 
+    def forward(self, *obervations):
```

Comment:
this is not a `*observations` because observations is known to only have one element. this is just `observations`

Reply:
fixed

Reply:
When I change the parameter to `observations` I got the error from pylint
`Parameters differ from overridden 'forward' method (arguments-differ)`

Reply:
yeah, this is a flaw in pytorch. you have to use a `# pylint disable=arguments-differ`